sshd-core/src/main/java/org/apache/sshd/client/ClientBuilder.java

```diff
@@ -79,7 +79,9 @@
                 BuiltinSignatures.nistp256,
                 BuiltinSignatures.nistp384,
                 BuiltinSignatures.nistp521,
+                BuiltinSignatures.sk_ecdsa_sha2_nistp256,
                 BuiltinSignatures.ed25519,
+                BuiltinSignatures.sk_ssh_ed25519,
```

Comment:
I am not comfortable to adding it as default for the client (for now) - users who want this should make an "effort" (at least for now).